components/sensor/xiaomi_ble.rst

```diff
@@ -365,6 +356,33 @@ Configuration example:
         illuminance:
           name: "MJYD02YL-A Illuminance"
 
+MCCGQ02HL
+*********
+
+Xiaomi Mijia Window Door Sensor 2 broadcasts light on/off status, door open/closed and battery status. Requires a bindkey in order to decrypt the received data (see :ref:`obtaining_the_bindkey`). Implemented as a hybrid sensor, needs both ``sensor`` and ``binary_sensor`` in config.
```

Comment:
> needs both ``sensor`` and ``binary_sensor`` in config.

Oh that should be done on the codegeneration side instead.

Please set

```python
AUTO_LOAD = ["sensor"]
```

in your new `binary_sensor.py` file, and then sensor will be automatically loaded.

Reply:
:heavy_check_mark: